enterprise/frontend/src/embedding-sdk/test/auth-flow.unit.spec.tsx

```diff
@@ -75,7 +90,7 @@ describe("SDK auth and init flow", () => {
     setupCurrentUserEndpoint(createMockUser({ id: 1 }));
 
     setupCardEndpoints(MOCK_CARD);
-    setupCardQueryEndpoints(MOCK_CARD, {} as any);
+    setupCardQueryEndpoints(MOCK_CARD, MOCK_DATASET);
```

Comment:
This mocked incorrectly which failed `getSensibleDisplays` function call which expects proper query data.